### PR TITLE
POLIO-912: make initial_org_unit required and refactor dialog tab to allow an icon

### PR DESCRIPTION
**Pull request by codeKameleon**
- Initial org unit field is necessary to create a campaign though it was not required
- It was not clear that you need at least dates filled on one round to access the Scope tab

Related JIRA tickets : [POLIO-912](https://bluesquare.atlassian.net/browse/POLIO-912?atlOrigin=eyJpIjoiN2RjM2Q5M2EzZjMxNGU4YThhYjEyMjlmNDA1NzM5MTIiLCJwIjoiaiJ9)

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [x] Are my typescript files well typed
- [ ] New translations have been added or updated if new string have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- add yup validation on initial_org_unit and make it required + add required prop to the Select input
- refactor dialog tab to allow an icon

## How to test

 - Go to Polio > Campaigns
 - Create a new campaign
 -  Scope tab should be disabled and an icon is displayed above the tab title. On hover you can see a popover showing info to enable the tab
 - Fill/ Select a value for required fields (Obr Name, initial region) and choose a start date and end date for at least one round to access Scope tab

## Print screen / video

https://user-images.githubusercontent.com/25134301/228619877-c3b07e02-8fb7-4d33-888f-02b97654dd2f.mov

[POLIO-912]: https://bluesquare.atlassian.net/browse/POLIO-912?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: plugins/polio/js/src/components/MainDialog/PolioDialogTab.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                (disabled && title === formatMessage(MESSAGES.scope) && (
+                    <Tooltip
+                        title={formatMessage(MESSAGES.scopeUnlockConditions)}
+                    >
+                        <InfoIcon
+                            fontSize="small"
+                            className={classes.tabIcon}
+                        />
+                    </Tooltip>
+                )) || <></>
```
Comment (quang-le) [inline plugins/polio/js/src/components/MainDialog/PolioDialogTab.tsx:62]: works with `|| undefined` as well

File: plugins/polio/js/src/styles/theme.js [hunk 2]
```diff
@@ -205,4 +205,24 @@ export const useStyles = makeStyles(() => ({
...
+        color: `${theme.palette.text.disabled} !important`,
+        cursor: 'default',
+    },
+    tabIcon: {
+        position: 'relative',
+        top: 1,
+        left: theme.spacing(0.5),
+        color: theme.palette.primary.main,
+        cursor: 'pointer',
+    },
```
Comment (quang-le) [inline plugins/polio/js/src/styles/theme.js:227]: :+1:

**Review by quang-le — CHANGES_REQUESTED** [review]
When I change tabs without filling out OBR and initial region, the tab errors, and the errored fields should be in error state, but only OBR name is in error:


https://user-images.githubusercontent.com/38907762/228807614-3314b579-8b9d-4366-829b-498a9f995653.mov

**Review by quang-le — APPROVED** [review]
I fixed the error state of initial org unit so it behaves like the rest of the form

**Review by hakifran — APPROVED** [review]
Seems good to me.

**Pull request merged**